src/octoprint/util/comm.py

```diff
@@ -2928,11 +2923,15 @@ def convert_line(line):
                         "SD init fail" in line
                         or "volume.init failed" in line
                         or "openRoot failed" in line
+                        or "SD Card unmounted" in line
+                        or "SD card released" in line
```

Comment:
Where did you find that string? I'm looking at the Marlin source but so far haven't found anything similar in there (doesn't even seem to have any string for umounting sadly).

Reply:
[here](https://github.com/prusa3d/Prusa-Firmware/blob/MK3/Firmware/cardreader.cpp#L267)

Reply:
Ah, Prusa Firmware, gotcha, thanks!